### ISSUE TITLE
ISSUE: Error during startup after reinstall

### ISSUE THREAD

**Issue by bcrooker** [moralmunky/Home-Assistant-Mail-And-Packages#205]
**Describe the bug**

**Environment (please complete the following information):**
 - OS: Docker
 - Type of device: Docker vs Synology
 - Home Assistant version: 0.113.3
 - Hassio/Docker/Core? Docker Core
 - Component version: [e.g. 0.1.2] 0.2.2-b21

**Logs**
Paste your error logs here.

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 179, in _async_setup_platform
    await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.8/asyncio/tasks.py", line 483, in wait_for
    return fut.result()
  File "/config/custom_components/mail_and_packages/sensor.py", line 71, in async_setup_entry
    sensors.append(PackagesSensor(data, variable))
  File "/config/custom_components/mail_and_packages/sensor.py", line 198, in __init__
    self.update()
  File "/config/custom_components/mail_and_packages/sensor.py", line 254, in update
    self.data.update()
  File "/usr/src/homeassistant/homeassistant/util/__init__.py", line 239, in wrapper
    result = method(*args, **kwargs)
  File "/config/custom_components/mail_and_packages/sensor.py", line 153, in update
    total = info[ATTR_COUNT] - data[delivered]
KeyError: 'usps_delivered'

**Screenshots**

**Additional context**
I started getting with the latest stable version, tried upgrading to beta to see if it addressed the issue.  I have also tried uninstalling, removing all files and reinstalling.  Config pulled from core.config_entities is:

            {
                "connection_class": "cloud_poll",
                "data": {
                    "folder": "\"Postal\"",
                    "generate_mp4": false,
                    "gif_duration": 5,
                    "host": "imap.gmail.com",
                    "image_path": "/config/images/mail_and_packages/",
                    "image_security": false,
                    "password": "OMITTED",
                    "port": 993,
                    "resources": [
                        "amazon_delivered",
                        "amazon_packages",
                        "fedex_delivered",
                        "fedex_delivering",
                        "fedex_packages",
                        "mail_updated",
                        "ups_delivered",
                        "ups_delivering",
                        "ups_packages",
                        "usps_mail",
                        "usps_delivering",
                        "usps_delivered",
                        "zpackages_transit",
                        "zpackages_delivered",
                        "usps_packages"
                    ],
                    "scan_interval": 5,
                    "username": "OMITTED"
                },
                "domain": "mail_and_packages",
                "entry_id": "16c6cf10025845fd86a6923db1de6e8e",
                "options": {
                    "folder": "\"Postal\"",
                    "generate_mp4": false,
                    "gif_duration": 5,
                    "host": "imap.gmail.com",
                    "image_path": "/config/images/mail_and_packages/",
                    "image_security": false,
                    "password": "OMITTED",
                    "port": 993,
                    "resources": [
                        "amazon_delivered",
                        "amazon_packages",
                        "fedex_delivered",
                        "fedex_delivering",
                        "fedex_packages",
                        "mail_updated",
                        "ups_delivered",
                        "ups_delivering",
                        "ups_packages",
                        "usps_mail",
                        "usps_delivering",
                        "usps_delivered",
                        "zpackages_transit",
                        "zpackages_delivered",
                        "usps_packages"
                    ],
                    "scan_interval": 5,
                    "username": "OMITTED"
                },
                "source": "user",
                "system_options": {
                    "disable_new_entities": false
                },
                "title": "imap.gmail.com",
                "unique_id": null,
                "version": 1
            }

**Comment by firstof9** [comment]
Please enable debugging and post your logs please.

**Comment by bcrooker** [comment]
[home-assistant.log](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/files/5065786/home-assistant.log)

See attached

**Comment by firstof9** [comment]
Did you happen to modify the `core.config_entities` file? The sensor list is out of order this is why you're getting the error.

**Comment by bcrooker** [comment]
Not within the section I posted above, no.  I did try removing the section entirely when uninstalling and reinstalling.  Or is there something outside of the section above that is incorrect?

**Comment by firstof9** [comment]
Try unchecking one of the sensors via Options and then go back and re-check it again.

**Comment by bcrooker** [comment]
i had tried that before, but tried it again just now with no change.  is there a file i can modify manually?

**Comment by firstof9** [comment]
In the `core.config_entities` if you swap the lines around
```
                    "usps_delivering",
                    "usps_delivered",
```
to 
```
                    "usps_delivered",
                    "usps_delivering",
```
That should fix it, still not sure how they are not in alphabetical order.

**Comment by bcrooker** [comment]
thanks - making that change worked!  strange - I tried uninstalling, wiping everything and reinstalling multiple times.  it seems relatively new for it to allow the selection of sensors, so I manually selected which ones I wanted (removing DHL and the Canada ones) - perhaps the latter was the issue?

**Comment by firstof9** [comment]
Home Assistant is supposed to store them in alphabetical order, I'm not sure what happened there.
At least you're back up and running.

**Comment by jes1417** [comment]
I'm having the same issue with all sensors not available with 0.3.0-b1 and 2 rolling back to b0 works for me you think this is the same issue?

**Comment by firstof9** [comment]
Could be, try b3 there was a typo that I've fixed.
